test/test_readme_sample.py

```diff
@@ -47,24 +49,42 @@ def print(*args):
 
         # Use Wine data set for training and test data
         feature_dim = 4  # dimension of each data point
-        _, training_input, test_input, _ = wine(training_size=12,
-                                                test_size=4,
+        # sample_train, training_input, test_input, class_labels
```

Comment:
This code, ie the body of the test from here ` # --- Exact copy of sample code ----------------------------------------` to the similar marker, is supposed to be a direct copy/paste of the sample code that is shown in the readme. The rationale being to make sure that the sample code we show continues to work.

We can change the readme to whatever sort of sample we thing might best showcase things and be a simple sample a user can run. Whatever it is though should just be copy/paste here in this test and the two should not diverge.

If you feel you need some other better/different test cases for VQC thats fine. This though was specific to making sure we test the readme sample that it continues to work.

Reply:
That's right, once this test works, I will update readme.